pdp/static/js/crmp_controls.js

```diff
@@ -136,8 +136,22 @@ function getResetButton(map) {
 }
 
 function getCRMPDateRange(map) {
-    var dateRange = getDateRange(true);
-    $('#from-date, #to-date', dateRange).change(pdp.curry(dateChange, map)).change(pdp.curry(CRMPFilterChange, map));
+    var calendar = calendars['gregorian'];
+    var units = 'days';
+    var cfTimeSystem = new calendars.CfTimeSystem(
+        units,
+        new calendars.CalendarDatetime(calendar, 1870, 1, 1),
+        Math.floor((2100 - 1870 + 1) * 365.2425)
+    );
+    var startDate = cfTimeSystem.firstCfDatetime();
+    var endDate = cfTimeSystem.todayAsCfDatetime();
+
```

Comment:
When I run this portal on my work station, the actual date range seems to be 1870-01-01 to 2100-01-01, not 1870-01-01 to today. It lets me set the end date to, say, 2050-06-21 without complain.

![daterange](https://user-images.githubusercontent.com/4512605/58666885-65e4e000-82e9-11e9-99d4-bd3ee9c5eed9.png)